packages/mds-agency/request-handlers.ts

```diff
@@ -197,15 +194,12 @@ export const updateVehicle = async (req: AgencyApiRequest, res: AgencyUpdateVehi
     } else {
       const device = await db.updateDevice(device_id, provider_id, update)
       // TODO should we warn instead of fail if the cache/stream doesn't work?
-      const { version } = res.locals
       try {
         await Promise.all([cache.writeDevice(device), stream.writeDevice(device)])
       } catch (error) {
         logger.warn(`Error writing to cache/stream ${error}`)
       }
-      return res.status(201).send({
-        version
-      })
+      return res.status(201).send({})
```

Comment:
Does `res.sendStatus(201)` work or does `res.status(201).send({})` result in different content type headers?

Reply:
It sends a different header. It takes out the `charset: utf-8`.